.github/workflows/firefox-ios-update-effective-tld-names-file.yml

```diff
@@ -43,6 +43,7 @@ jobs:
           echo "create_pr=true" >> $GITHUB_ENV
           cp tmp.dat firefox-ios/Shared/effective_tld_names.dat
           echo "branch_name=refactor-update-effective-tld-names" >> $GITHUB_ENV
+          echo "pr_title=Refactor [vXXX] Update effective_tld_names file $current_date" >> $GITHUB_ENV
```

Comment:
It would be nice if we can fill in the version as we do for the AS PR .. https://github.com/mozilla-mobile/firefox-ios/blob/476a828da38ff5c3ce3ab0edabbe7b5dcfac3952/.github/workflows/check-rust-component-dependency.yml#L81